azureiot.sln

```diff
@@ -95,7 +95,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Shared.Tests", "shared\tests\Microsoft.Azure.Devices.Shared.Tests.csproj", "{CEEE435F-32FC-4DE5-8735-90F6AC950A01}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.PnpServiceSample", "iothub\service\samples\PnpServiceSamples\Thermostat\Microsoft.Azure.Devices.PnpServiceSample.csproj", "{5B7C14D6-1441-489B-B9FB-1EEEA43B8C48}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Thermostat", "iothub\service\samples\PnpServiceSamples\SimpleThermostat\Thermostat.csproj", "{08A383A4-9FA3-4585-A745-57A7CDBAB9DC}"
```

Comment:
The main solution should not be updated. This service client sample needs to live as it's own sample.

Reply:
For reference: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/iothub/device/samples/PnpDeviceSamples

Reply:
why should we be creating sample solution, shouldn't it be just a csproj?

Reply:
Everything that you put in `azureiot.sln` will show up when you launch the solution. These samples are not required to be loaded when you launch the solution for the source code.

Reply:
synced offline on this, we are going with this approach of samples being part of main solution